### PR TITLE
Remove core security job

### PR DESCRIPTION
**Pull request by olblak**
This PR update security release documentation and remove the security release build trigger job

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -336,23 +336,39 @@ You can re-trigger individually the two downstream jobs, release and packaging.
 ====
 
 === Security
-The security release follows the same process as the stable one except that it also promotes git commits from a private repository to the public one and also promotes maven artifacts from a private maven repository to the public one.
+The security release follows the same process as the stable one excepted that artifacts are published in private location. So we need promote git commits from a private repository to the public one then promotes maven artifacts from a private maven repository to the public one.
```
Comment (timja) [inline README.adoc:339]:
```suggestion
The security release follows the same process as the stable one except that artifacts are published in private. So we need promote git commits from a private repository to the public one then promote maven artifacts from a private maven repository to the public one.
```

File: README.adoc [hunk 2]
```diff
@@ -336,23 +336,39 @@ You can re-trigger individually the two downstream jobs, release and packaging.
...
 
 . Prepare jenkinsci-cert/jenkins repository -> missing documentation link
-. Create a branch on jenkins-infra/release with a branch name that match the release branch from jenkinsci/jenkins like `stable-2.135`
+. Create a branch on jenkins-infra/release with a branch name that match the release branch from jenkinsci/jenkins like `security-stable-2.135`
```
Comment (timja) [inline README.adoc:342]:
```suggestion
. Create a branch on jenkins-infra/release with a branch name that match the release branch from jenkins-cert/jenkins like `security-stable-2.135`
```

File: README.adoc [hunk 3]
```diff
@@ -336,23 +336,39 @@ You can re-trigger individually the two downstream jobs, release and packaging.
...
-. Trigger the security link:https://release.ci.jenkins.io/blue/organizations/jenkins/core%2Fsecurity%2Frelease/branches/[job]
+. Trigger the generic Release link:https://release.ci.jenkins.io/job/core/job/release/[job] from the appropriated branch like `security-stable-2.235`
+.. Force repository scan
+.. Trigger the first build to have access to job parameter
+.. Trigger a job with the correct parameters
+... `RELEASE_PROFILE` set to `security`
+... `RELEASE_GIT_BRANCH` set to `unused` as we already define it in the release profile file, which overrides the job parameter
+... `MAVEN_REPOSITORY_NAME` set to `unused` as we already define it in the release profile file, which overrides the job parameter
+... `VALIDATION_ENABLED` set to true in to have a validation stage
+.. Trigger the generic Packaging job link:https://release.ci.jenkins.io/job/core/job/package/[job] from the appropriated branch like `security-stable-2.235` with correct parameters
```
Comment (timja) [inline README.adoc:356]:
```suggestion
.. Trigger the generic Packaging job link:https://release.ci.jenkins.io/job/core/job/package/[job] from the appropriate branch like `security-stable-2.235` with correct parameters
```

File: README.adoc (hunk 1 above)
Comment (olblak) [reply README.adoc:339]: Thanks, one day I'll be good at writing (english) documentation

File: README.adoc [hunk 4]
```diff
@@ -336,23 +336,39 @@ You can re-trigger individually the two downstream jobs, release and packaging.
 ====
 
 === Security
-The security release follows the same process as the stable one except that it also promotes git commits from a private repository to the public one and also promotes maven artifacts from a private maven repository to the public one.
+The security release follows the same process as the stable one except that artifacts are published in private. So we need promote git commits from a private repository to the public one then promote maven artifacts from a private maven repository to the public one.
```
Comment (daniel-beck) [inline README.adoc:339]:
```suggestion
The security release follows the same process as the stable one except that artifacts are published in private. So we need to promote git commits from a private repository to the public one then promote maven artifacts from a private maven repository to the public one.
```

No sentence-per-line formatting 😦

File: README.adoc [hunk 5]
```diff
@@ -336,23 +336,39 @@ You can re-trigger individually the two downstream jobs, release and packaging.
...
-.. `RELEASE_GIT_BRANCH` set to the `jenkinsci-cert/jenkins` release branch like `stable-2.135`
+.. `RELEASE_GIT_BRANCH` set to the `jenkinsci-cert/jenkins` release branch like `security-stable-2.135`
 .. `JENKINS_VERSION` set to the final release version that will be packaged. If set to 'stable' or 'weekly' then the packaging job will try to guess the version based on what was pushed to the maven repository. cfr settings.
 .. `PACKAGING_GIT_BRANCH` set to the appropriated jenkinsci/packaging branch
 .. `MAVEN_REPOSITORY_NAME` set to the maven repository name where we are going to publish staging maven artifacts. This is also the source location used by the packaging job to build distribution packages
-. Trigger the security link:https://release.ci.jenkins.io/blue/organizations/jenkins/core%2Fsecurity%2Frelease/branches/[job]
+. Trigger the generic Release link:https://release.ci.jenkins.io/job/core/job/release/[job] from the appropriated branch like `security-stable-2.235`
+.. Force repository scan
+.. Trigger the first build to have access to job parameter
```
Comment (daniel-beck) [inline README.adoc:350]:
```suggestion
.. Trigger the first build to have access to job parameter and immediately abort it
```

**Pull request merged**